zerver/lib/users.py

```diff
@@ -24,6 +24,19 @@ def check_short_name(short_name_raw: Text) -> Text:
         raise JsonableError(_("Bad name or username"))
     return short_name
 
+def check_bots_allowed_for_non_admins(user_profile: UserProfile, bot_type: int) -> None:
+    """Raises an exception if the users cannot add a bot
+    in their organization."""
+    # Realm administrators can always add bot
+    if user_profile.is_realm_admin or\
```

Comment:
Stylistically, we always include a space between the `or` and the `\` here.  I think you do this one other place in the code too.

I usually just use parens for this line-wrapping.

That said, I think this code would actually be better with the comment two lines above removed, and having two `if` statements that each return (basically because that's more readable and allows our line-level code coverage checks to very we have tests for each case).